src/cmd/d9.c

```diff
@@ -824,6 +826,7 @@ usage (void)
 "   -u,--uid              authenticate as uid (default is your euid)\n"
 "   -t,--timeout SECS     give up after specified seconds\n"
 "   -p,--privport         connect from a privileged port (root user only)\n"
+"   -t,--trace            trace 9P protocol on stderr\n"
```

Comment:
`-t` used in both `--trace` and `--timeout` in the usage message. It looks like `-t, --timeout` just needs to be removed.